swift_browser_ui_frontend/src/components/ObjectTable.vue

```diff
@@ -6,6 +6,7 @@
       class="groupControls"
     >
       <b-select
+        data-testid="bucketsPerPage"
```

Comment:
Our linting rules state that `vue` attributes should precede ordinary ones, this raises linter warnings when running eslint upon `npm run build-devel`, and can be automatically fixed by running `npm run lint`

Reply:
done